mreuter/CSC_Testing/mt_utils.py

```diff
@@ -0,0 +1,122 @@
+from astropy.coordinates import AltAz, ICRS, EarthLocation, Angle
+from astropy.time import Time
+import astropy.units as u
+
+from lsst.ts import salobj
+
+def altaz_to_radec(alt, az):
+    location = EarthLocation.from_geodetic(
+        lon=-70.747698 * u.deg, lat=-30.244728 * u.deg, height=2663.0 * u.m
+    )
+
+    current_time = salobj.astropy_time_from_tai_unix(salobj.current_tai())
+    current_time.location = location
+
+    coord_frame_radec = ICRS()
+
+    alt_az = AltAz(
+        az=Angle(az, unit=u.deg),
+        alt=Angle(alt, unit=u.deg),
+        location=location,
+        obstime=current_time
+    )
+
+    return alt_az.transform_to(coord_frame_radec)
+
+async def slew_to_park(mtcs, rotpa=0.0):
+    ra_dec = altaz_to_radec(80.0, 0.0)
+
+    mtcs.rem.mtptg.cmd_raDecTarget.set(
+        ra=ra_dec.ra.hour,
+        declination=ra_dec.dec.deg,
+        rotPA=Angle(rotpa, unit=u.deg).deg,
+        targetName="Park position",
+        frame=2,
+        epoch=2000,
+        equinox=2000,
+        parallax=0,
+        pmRA=0,
+        pmDec=0,
+        rv=0,
+        dRA=0,
+        dDec=0,
+        rotFrame=1,
+        rotMode=1,
+    )
+
+    await slew(mtcs, True)
+
+async def slew_to_flatfield(mtcs, rotpa=-180.0):
+    ra_dec = altaz_to_radec(39.0, 205.7)
```

Comment:
I think this coordinate should either be in a config file, or maybe defined at the top and not buried in the code?